services/report/parser/__init__.py

```diff
@@ -11,7 +11,7 @@ def get_proper_parser(upload: Upload, contents: bytes):
         if contents.startswith(b"{") and contents.endswith(b"}"):
             return VersionOneReportParser()
         else:
-            with sentry_sdk.configure_scope() as scope:
+            with sentry_sdk.new_scope() as scope:
                 scope.set_extra("upload_extras", upload.upload_extras)
                 scope.set_extra("contents", contents[:64])
                 sentry_sdk.capture_message("Upload `format_version` lied to us")
```

Comment:
```suggestion
            scope = sentry_sdk.Scope.get_current_scope():
            scope.set_extra("upload_extras", upload.upload_extras)
            scope.set_extra("contents", contents[:64])
            sentry_sdk.capture_message("Upload `format_version` lied to us")
```

Reply:
hm, does this really do the right thing? won’t that leak the `extra`s outside of this function?

Reply:
If the scope is really only meant to be used for the one `capture_message`, the `new_scope` you had there before is the better way to go about this -- sorry for the confusion!

Reply:
thanks for the clarification, I reverted the change back.